fbrp/examples/10_ifttt/ifttt.py

```diff
@@ -0,0 +1,19 @@
+import a0
+import signal
+import requests
+
+EVENT_NAME = a0.cfg(a0.env.topic(), "/ifttt/event_name", str)
+KEY = a0.cfg(a0.env.topic(), "/ifttt/key", str)
+EVENT_TOPIC = a0.cfg(a0.env.topic(), "/event_topic", str)
+a0.update_configs()
+
+
+def call_robo_crit_event(pkt):
+    print(f"Pinging IFTTT Event {EVENT_NAME}")
+    requests.post(f"https://maker.ifttt.com/trigger/{EVENT_NAME}/with/key/{KEY}")
+
+
+s = a0.Subscriber(f"{EVENT_TOPIC}", lambda pkt: call_robo_crit_event(pkt))
+
+# a0.update_configs()
```

Comment:
Remove dead code.
If you want to include `a0.update_configs()` somewhere, put it into the `call_robo_crit_event` function.

Reply:
Note: It wont change the EVENT_TOPIC used by the Subscriber.

Reply:
Thanks for pointing this out: Wouldn't we want `s` to change subscription though if we are permitting online update of parameters? Is there a better way of doing this?

Reply:
Uhh... ok. I don't necessarily think we should support changing all the variables. I think the code complexity will increase faster than the usefulness.

The following is an example of how you would do it, if you really wanted:
```py
class Foo:
    def __init__(self):
        self._topic: str = None
        self._s: a0.Subscriber = None

    def on_cfg_change(self, pkt):
        cfg = json.loads(pkt.payload)
        if self._topic != cfg.get("event_topic"):
            self._topic = cfg.get("event_topic")
            self._s = None
            if self._topic:
                self._s = a0.Subscriber(self._topic, call_robo_crit_event)


watcher = a0.CfgWatcher(a0.env.topic(), Foo().on_cfg_change)
```

`CfgWatcher` is not smart enough to watch a sub-field.
None of the code here was tested, at all.